test-e2e/har/har.test.ts

```diff
@@ -0,0 +1,193 @@
+import 'mocha';
+import * as assert from 'assert';
+import { parseHttpArchiveText, compareEntries, isEquivalent } from './har'
+import * as fs from 'fs';
+import * as path from 'path';
+
+describe('experiment', () => {
+  it('should only care about attributes a and b', () => {
+    let model = {
```

Comment:
should we use `const` instead of `let`?

Reply:
This 'experiment' shouldn't have been committed; will remove